### ISSUE TITLE
Login Integration: uauth.user() fails due to missing username

### ISSUE THREAD

**Issue by calebrosario** [unstoppabledomains/uauth#100]
@web3-react v6
@uauth/js v2.4.0

Login Integration
TLDR: username is set a an optional value but breaks when not set and attempting to get UserInfo

`await activate(uauth);`
do the redirect, approve and sign via metamask
user is redirected to correct authorize route
in this route we run `uauth.user()`
https://github.com/unstoppabledomains/uauth/blob/52618c8ba7861f39ecb968d3524950597c72b704/packages/js/src/Client.ts#L422

to then call `authorization()`

https://github.com/unstoppabledomains/uauth/blob/52618c8ba7861f39ecb968d3524950597c72b704/packages/js/src/Client.ts#L416

Here we see that the interface `AuthorizationOptions` username as optional

https://github.com/unstoppabledomains/uauth/blob/52618c8ba7861f39ecb968d3524950597c72b704/packages/js/src/types.ts#L155

but here we see that `username` is verified as a required field using the `_getUsername` to verify one exist.

https://github.com/unstoppabledomains/uauth/blob/52618c8ba7861f39ecb968d3524950597c72b704/packages/js/src/ClientStore.ts#L134
